projects/components/src/utils/test/datagrid/datagrid.wo.ts

```diff
@@ -32,6 +32,26 @@ export class ClrDatagridWidgetObject extends WidgetObject<ClrDatagrid> {
         return this.getNodeText(this.getCell(row, column));
     }
 
+    /**
+     * Gives the text content of all the cells in a row.
+     */
+    getRowValues(rowIndex: number): string[] {
+        const toReturn = [];
```

Comment:
Couldn't we use
```typescript
return getTexts(`${ROW_TAG}:nth-of-type(${row + 1}) ${CELL_TAG}`)
```
Please document `rowIndex`, at least to specify 0 or 1 based. See